fec.config.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
    * Add additional webpack plugins
    */
   plugins: [],
+  chromeHost: process.env.FEC_CHROME_HOST ?? undefined,
```

Comment:
This is not necessary. These variables need to be defined if running cypress e2e tests which this repo is not.

Reply:
removed